crates/rooch-framework/sources/timestamp.move

```diff
@@ -0,0 +1,72 @@
+/// This module keeps a global wall clock that stores the current Unix time in microseconds.
+/// It interacts with the other modules in the following ways:
+/// * genesis: to initialize the timestamp
+/// * L1 block: update the timestamp via L1s block header timestamp
+/// * TickTransaction: update the timestamp via the time offset in the TickTransaction(TODO)
```

Comment:
Plan to provide second-level accuracy through TickTransaction？

Reply:
yes